### DISCUSSION TITLE
Need example pages

### DISCUSSION THREAD

**Discussion by adiramardiani** [uikit/uikit#4864] [General]
Hi @saschadube  and all uikit team, After all this time I know uikit, and for several years in frontend I break for a while
Yes, uikit (version 2) is one of my favorites, and when I see version 3 (although it's still RC) it looks very promising :heart_eyes:  .

With many components that are ready to use, a simple display, fresh and modern look, and the impornant think is usefull, I think this will be better than version 2

What happened when in version 2, with so many components in the documentation, I had a hard time implementing it cause maybe at that time I didn't really understand uikit. Yes, for experts and professionals it is very easy to combine and change the styling of each component, but without official examples, beginners will be very difficult to combine and styling for each component. I know many way to combine and use all feature, but if have **Example Page** to every component used, is very big improve.

My request is, beside pro, documentation or changelog menu. Will be amazing if have a example menu. Like blog page who use : navigation, offcanvas, card, comment component
Like landing page who use : navigation, offcanvas, slider, video, paralax component
Like dashboard page who usel : navigation, table list, edit form
etc

This is very usefull for beginners, after read documentation for some component (now documentation and examples code per component is very good) and he want to combine each component to implement in some page, he confused how to styling component to show what he wants. I knowm Maybe for implementation many way to do this, every people, every person have a way to use this framework, but if beginners see the best practice to use/custom/implement this component framework in creator way and perspective, this will be AMAZING :+1:

**Comment by mattfletcher** [comment]
https://github.com/zzseba78/Kick-Off/blob/master/README.md

and

https://getuikit.com/assets/uikit/tests/index.html

Are two resources I've found very helpful in the past.

**Comment by adiramardiani** [comment]
> Are two resources I've found very helpful in the past.

Yes a see Kick-Off before, and it's good, but I mean : 

> ... but without **official examples** ...

Maybe something like that (kick-off samples page), but build by official uikit team
So the example page is on the top menu uikit official website, (like) beside : Pro | Documentation | Changelog, no in 3rd party website
Not funny if community or 3rd party have a good sample pages, and uikit it self is no :)

**Comment by rangka-kacang** [comment]
I beg to differ.

As we all know, this is a front-end framework and not a ready-made template.

As far as I'm concern, the use case scenario is totally subjective and up to the authors' creativity to implement the way they prefer. I don't think it is necessary for the "official team" to provide such examples to avoid the community being "mainstream" or be made as "guideline".

Take this (purely UIkit 3) "dashboard" and look at the sidebar menu for instance:
1. https://zzseba78.github.io/Kick-Off/dashboard.html
2. https://codepen.io/rangka_kacang/pen/Yadrza

As you can see, with the power of the core framework itself - one can easily create whatever they want easily. It's all in your imagination (and a little research). It is really fun! Don't follow examples, create your own solutions to your problems and you will appreciate UIkit 3 more, I guarantee...

Good luck!

**Comment by Axelaredz** [comment]
and https://codepen.io/collection/AevxaJ/
